access-modifier-checker/src/main/java/org/kohsuke/accmod/impl/Checker.java

```diff
@@ -420,6 +422,11 @@ public void visitTypeInsn(int opcode, String type) {
         @Override
         public void visitMethodInsn(int opcode, String owner, String name, String desc, boolean itf) {
             log.debug(String.format("Visiting method %s#%s", owner, name));
+
+            if (currentClass.equals(owner)) {
```

Comment:
I suspect this suffers from the same limitation as the previous attempt: https://github.com/kohsuke/access-modifier/pull/6#discussion_r117833411

Reply:
Thanks, I was not aware of #6. I will take a look at this tomorrow.